### PR TITLE
Add documentation for action icons and action divider

### PR DESCRIPTION
**Pull request by gabrielgiroe1**
Screenshot 2024-04-03 at 18 48 56

### CODE REVIEW

File: docs/3.0/customizable-controls.md [hunk 1]
```diff
@@ -163,6 +163,8 @@ actions_list exclude: DisableAccount
 # Or
 actions_list exclude: [ExportSelection, PublishPost]
 ```
+::::info
+The actions icon and action divider will defined in def action. Here is an example of [action icon and divider](actions.md#action_icons_and_action_divider)
```
Comment (Paul-Bob) [inline docs/3.0/customizable-controls.md:167]:
```suggestion
The actions icon and dividers are defined in `def actions` method. Here is an example of [action icon and divider](actions.md#action_icons_and_action_divider)
```

**Comment by Paul-Bob** [comment]
Closing in favor of https://github.com/avo-hq/avodocs/pull/187 ( couldn't commit on that PR )

File: docs/3.0/actions.md [hunk 2]
```diff
@@ -362,6 +362,47 @@ end
 ```
 :::
 
+:::option `action icons`
```
Comment (adrianthedev) [inline docs/3.0/actions.md:365]: Other labels have only the option name. Theis would be `icon`

Comment (Paul-Bob) [reply docs/3.0/actions.md:365]: @adrianthedev the other options on actions are response types I changed the icon an divider location in this PR https://github.com/avo-hq/avodocs/pull/187

I couldn't commit to this one

Comment (adrianthedev) [reply docs/3.0/actions.md:365]: I get that, but I think we should do the same with these arguments.

Maybe in a docs reorganization?

Comment (Paul-Bob) [reply docs/3.0/actions.md:365]: Yes agree, just created this issue to track it https://github.com/avo-hq/avodocs/issues/195

**Pull request closed without merging**